x-pack/plugins/security_solution/common/endpoint/models/policy_config_helpers.ts

```diff
@@ -35,7 +35,11 @@ const disableCommonProtections = (policy: PolicyConfig) => {
 
   for (const key in policyOutput) {
     if (Object.prototype.hasOwnProperty.call(policyOutput, key)) {
-      const os = key as keyof PolicyConfig;
+      if (key === 'meta') {
```

Comment:
need to filter out meta otherwise, the function will be looking for keys specific to OSes. There may be a better way to do this without the continue line - open to suggestions

Reply:
could change the arg type on line 33 to `Omit<PolicyConfig, 'meta'>`. then when calling `disableCommonProtections` on line 18, we can just filter it out with lodash `omit` or reduce. this saves needing to update the os type. though a bit less efficient (negligibly).

alternatively, if your main concern is the `continue` for linter, can just convert `disableCommonProtections` logic to a reduce.